nvflare/fuel/hci/client/fl_admin_api_spec.py

```diff
@@ -77,26 +77,28 @@ def check_status(self, target_type: TargetType, targets: Optional[List[str]] = N
         pass
 
     @abstractmethod
-    def set_run_number(self, run_number: int) -> FLAdminAPIResponse:
-        """Sets a current run number.
+    def delete_run(self, run_number: str) -> FLAdminAPIResponse:
+        """Deletes a specified run.
+
+        This deletes the run folder corresponding to the run number on the server and
+        all connected clients. This is not reversible.
 
         Args:
-            run_number: run number in order to set for the current experiment, must be integer greater than 0
+            run_number (str): run number to delete
```

Comment:
If run_number is not a number anymore,
I think we should change it.

Maybe something like "run_name"/"run_id"?